packages/store/src/symbols.ts

```diff
@@ -144,6 +150,13 @@ export interface NgxsOnInit {
   ngxsOnInit(ctx?: StateContext<any>): void | any;
 }
 
+/**
+ * On change interface
+ */
+export interface NgxsOnChanges {
+  ngxsOnChanges(change?: NgxsSimpleChange): void;
```

Comment:
The '?' in this method declaration would mean that this method could be invoked without a value for the `change` parameter. This is not the case because it is always provided by the framework.
Please remove the `?`.

Reply:
I do not need, in this case, an argument

```
class State implements NgxsOnChanges {
  public ngxsOnChanges(): void {
     this.myService.trigger();
  }
}
```

Reply:
Yes, but that is not what the question mark means. You can always leave out parameters in your implementation and TS will be happy. Try it out.

Reply:
done